src/app/features/auth/auth.routes.ts

```diff
@@ -0,0 +1,14 @@
+import { Route } from '@angular/router';
+import { LoginComponent } from "./login/login.component";
+
+export const routes: Route[] = [
+  {
+    path: '',
+    redirectTo: '/login',
+    pathMatch: 'full'
+  },
```

Comment:
This automatically and alway redicrects user to login screen. What if the user is already logged in?